server.js

```diff
@@ -48,6 +49,7 @@ function getLocation(city) {
 let cache = {};
 function cacheLocation(city, data) {
   const location = new Location(data.results[0]);
```

Comment:
It looks like you're trying to create an instance, but are you calling the constructor correctly?

Reply:
I have to add another argument when I calling the constructor function
I will solve it